### ISSUE TITLE
Attempting to create an org named `UBID` results in a 500 error

### ISSUE THREAD

**Issue by axelstudios** [SEED-platform/seed#4714]
**Expected Behavior**
Creating a new organization with a unique name should work

**Actual Behavior**
If the new organization is named `UBID` it will create the org but fail with a 500 error: `This display name is already an access level name and cannot be used.`

**Steps to Reproduce**
1. Attempt to create a new organization named `UBID`

**Comment by kflemin** [comment]
I think this is the case for any field name. @haneslinger, I think you've already fixed something related to this in the past?

**Comment by axelstudios** [comment]
> I think this is the case for any field name. @haneslinger, I think you've already fixed something related to this in the past?

Yeah, this is happening because `access_level_names = ['UBID']`, and it's hitting unhandled errors when trying to create the TaxLotState UBID column

**Comment by axelstudios** [comment]
Seems like there's _another_ bug here as well - the Access Levels page has restrictions so that you can't use the display name of an existing column for the access levels. _But_, it does allow you to use the underlying name (e.g. `pm_property_id`) which then prevents you from importing any data into that column going forward
